config/hsc/fgcmCalibrateTractTable.py

```diff
@@ -0,0 +1,18 @@
+import os.path
+
+config.fgcmBuildStars.load(os.path.join(os.path.dirname(__file__),
+                                        'fgcmBuildStarsTable.py'))
+config.fgcmFitCycle.load(os.path.join(os.path.dirname(__file__),
+                                      'fgcmFitCycle.py'))
+config.fgcmOutputProducts.load(os.path.join(os.path.dirname(__file__),
+                                            'fgcmOutputProducts.py'))
+config.fgcmFitCycle.aperCorrFitNBins = 0
+config.fgcmFitCycle.useRepeatabilityForExpGrayCutsDict = {'N387': True,
```

Comment:
Is this "all filters True" or "just these specific filters True"? It's not clear from how the list is made: maybe there's a better way to make that clear? I don't know if using the `FilterDefinition`s in `hscFilters.py` might help?